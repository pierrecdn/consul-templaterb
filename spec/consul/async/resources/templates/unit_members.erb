<%= agent_members.map do |a|
      "#{a['Name']}/#{a['Addr']}/#{a.status}/#{a['Status']}"
    end.join("\n") %>