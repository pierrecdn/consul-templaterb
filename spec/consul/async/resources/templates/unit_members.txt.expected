tableaunode005-par/10.236.127.20/alive/1
kubes02e171-par/10.236.134.42/failed/4